library/swarm

```diff
@@ -0,0 +1,4 @@
+# maintainer: Andrea Luzzardi <al@docker.com> (@aluzzardi)
+# maintainer: Victor Vieux <vieux@docker.com> (@vieux)
+
+latest: git://github.com/docker/swarm-library-image@c0f8204b2a1bb34233cd2144df51a4de9572440c
```

Comment:
I guess what I'm getting at is that when `0.1.1` comes out, you won't still have `0.1.0` in this file either, since it won't be officially supported anymore.  So my question is, what's the harm in tagging this release too?